### PR TITLE
Adding suggestion images to configmap

### PR DESCRIPTION
**Pull request by johnugeorge**
Suggestion image mappings are populated in ConfigMap so that right suggestion algorithm services are spawned for each experiment.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/754)

### CODE REVIEW

File: manifests/v1alpha3/katib-controller/katib-config.yaml [hunk 1]
```diff
@@ -15,4 +15,19 @@ data:
       "TensorFlowEvent": {
         "image": "gcr.io/kubeflow-images-public/katib/v1alpha3/tfevent-metricscollector"
       }
+    },
```
Comment (gaocegege) [inline manifests/v1alpha3/katib-controller/katib-config.yaml]: Should we have a `,` here?

Comment (hougangliu) [reply manifests/v1alpha3/katib-controller/katib-config.yaml]: I don't think we should add ","

Comment (johnugeorge) [reply manifests/v1alpha3/katib-controller/katib-config.yaml]: Removed

**Comment by johnugeorge** [comment]
@hougangliu @gaocegege 
New var `DefaultKatibNamespace` is added which will be env value of `DefaultKatibNamespaceEnvName`. If not set, default value of `kubeflow` is chosen.

**Pull request merged**